ethereum/contracts/TablelandRigs.sol

```diff
@@ -524,6 +527,26 @@ contract TablelandRigs is
         return _getTokenURI(_toString(tokenId));
     }
 
+    /**
+     * @dev See {IERC721Metadata-safeTransferWhileFlying}.
+     */
+    function safeTransferWhileFlying(
+        address from,
+        address to,
+        uint256 tokenId
+    ) external {
+        // Verify `msg.sender` is the token owner (prevent transfers by approved address or operator)
+        if (ownerOf(tokenId) != _msgSenderERC721A())
+            revert ITablelandRigPilots.Unauthorized();
+        // Temporaritly set the transfer flag to allow a transfer to occur
+        _allowTransferWhileFlying = 1;
```

Comment:
Instead of values `0` and `1` for `_allowTransferWhileFlying`, the Moonbirds contract uses `1` and `2`. I can't think of why, unless it was just preference...

The one thing to note is that `_allowTransferWhileFlying` isn't initialized with any value, so it defaults to `0`. I don't think this has any implications, besides, _maybe_ a little extra gas (like 17 gas) upon each `safeTransferWhileFlying` call.

Reply:
Yeah, I don't get it...and using a `bool` actually decreases gas a bit.

Reply:
May as well do that then?

Reply:
Definitely -- added it in the latest!